instrumentation/oracle-ucp-11.2/testing/src/main/java/io/opentelemetry/instrumentation/oracleucp/AbstractOracleUcpInstrumentationTest.java

```diff
@@ -36,17 +42,42 @@ public abstract class AbstractOracleUcpInstrumentationTest {
   protected abstract void shutdown(PoolDataSource connectionPool) throws Exception;
 
   @BeforeAll
-  static void setUpMocks() throws SQLException {
-    MockDriver.register();
+  static void setUp() {
+    // This docker image does not work on arm mac. To run this test on arm mac read
+    // https://blog.jdriven.com/2022/07/running-oracle-xe-with-testcontainers-on-apple-silicon/
+    // install colima with brew install colima
+    // colima start --arch x86_64 --memory 4
+    // export TESTCONTAINERS_DOCKER_SOCKET_OVERRIDE=/var/run/docker.sock
+    // export DOCKER_HOST="unix://${HOME}/.colima/docker.sock"
+    String dockerHost = System.getenv("DOCKER_HOST");
+    if (!"aarch64".equals(System.getProperty("os.arch"))
```

Comment:
should we limit condition to mac arm specifically?

Reply:
not really, it won't work on any arm